src/basics.jl

```diff
@@ -1,3 +1,4 @@
+import Compat: String, take!
```

Comment:
will need to change https://github.com/JuliaWeb/Mux.jl/blob/c83c0572f1e0adcba20f7d46b6a9d48e96b60616/REQUIRE#L8 to 0.9.5 for `take!`